lib/looker-sdk/error.rb

```diff
@@ -76,6 +76,40 @@ def message
       response_message
     end
 
+    # Error Doc URL
+    #
+    # @return [String]
+    def error_doc_url(documentation_url)
+      return nil unless documentation_url
+      regexp = Regexp.new("https://(?<redirector>docs\.looker\.com\|cloud\.google\.com/looker/docs)/r/err/(?<api_version>.*)/(?<status_code>\\d{3})(?<api_path>.*)", Regexp::IGNORECASE)
+      match_data = regexp.match documentation_url
+      return nil unless match_data
+
+      key = "#{match_data[:status_code]}#{match_data[:api_path].gsub(/\/:([^\/]+)/,"/{\\1}")}"
+      error_doc = error_docs[key] || error_docs[match_data[:status_code]]
+      return nil unless error_doc
+
+      return "https://marketplace-api.looker.com/errorcodes/#{error_doc[:url]}"
+    end
+
+    def error_docs
+      @error_docs ||=
+        begin
+          sawyer_options = {
+            :links_parser => Sawyer::LinkParsers::Simple.new,
+            :serializer  => LookerSDK::Client::Serializer.new(JSON),
+            :faraday => Faraday.new
+          }
+
+          agent = Sawyer::Agent.new("https://marketplace-api.looker.com", sawyer_options) do |http|
```

Comment:
this url will be changing in the first half of the year, but we can worry about that when it does!